CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ For experimental package changes, see the [experimental CHANGELOG](experimental/
 
 ### :rocket: (Enhancement)
 
+* feat(sdk-metrics): exponential histogram export [#3506](https://github.com/open-telemetry/opentelemetry-js/pull/3506) @mwear
 * feat(sdk-metrics): add exponential histogram accumulation / aggregator [#3505](https://github.com/open-telemetry/opentelemetry-js/pull/3505) @mwear
```

Comment:
Can probably combine these changelog entries into one more user-friendly one that references both PRs since this release actually makes this feature available.